.github/workflows/ci.yml

```diff
@@ -0,0 +1,43 @@
+name: CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  main:
+    runs-on: ubuntu-latest
+    env:
+      GO_VERSION: '1.20'
```

Comment:
Why not use go 1.21 now ?

Reply:
The module itself cannot be 1.21 because of golangci-lint is 1.20.
https://go.dev/blog/toolchain

And I wanted to make CI consistent with project (for reproducible results).